src/zmq/zmqnotificationinterface.cpp

```diff
@@ -179,3 +181,37 @@ void CZMQNotificationInterface::NotifyTransactionLock(const CTransaction &tx)
         }
     }
 }
+
+void CZMQNotificationInterface::NotifyGovernanceVote(const CGovernanceVote &vote)
+{
+    for (std::list<CZMQAbstractNotifier*>::iterator i = notifiers.begin(); i != notifiers.end(); )
```

Comment:
The iterator initialization is _probably_ ok, but would prefer if we can move to using `auto` instead.

The convention in our codebase is to call iterators `it`, so would prefer than name to just `i`.

Reply:
as in `for (auto it = notifiers.begin(); it != notifiers.end(); )` ? and then change the others to `it` to

Reply:
Yep, this looks correct.

Reply:
please view cb16cf07